workshops/conditions_and_loops.py

```diff
@@ -0,0 +1,8 @@
+def solve_the_knight_move_problem(input_string):
+    x1, y1, x2, y2 = map(int, input_string.split())
+    if ((abs(x1-x2) == 1 and abs(y1-y2) == 2) or
+            (abs(x1-x2) == 2 and abs(y1-y2) == 1)):
+        return "Yes, it can"
```

Comment:
I want problem solutions to be relatively "pure", which in this case would mean a function that return True or False. Mixing input-output and the solution in the same function is usually not a good idea.

My current solution of having a small wrapper function isn't that great, either. Not sure how to change the design here, but the final version still should have a pure solution function and a wrapper.

So for now please separate the output message from the solution. Either by moving the message into a wrapper function or by removing the message and instead just returning True/False.